core/forms/use_restriction.py

```diff
@@ -31,8 +31,10 @@ def clean(self):
         cleaned_data = super().clean()
         restriction_class = cleaned_data.get('restriction_class')
         notes = cleaned_data.get('notes')
-        if not restriction_class and notes:
+        if not restriction_class:
             self.add_error('restriction_class', 'Please select a valid restriction class')
+        if not notes:
```

Comment:
We shall delete "not null" constraint on `notes` - I guess it is not a requirement and also many existing data declarations already have `notes` field empty -> to avoid errors on `edit`, we would need some kind of data migration.

Reply:
null constraint in clean() removed